.github/workflows/test.yml

```diff
@@ -11,31 +11,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ckan-version: [ckan-2.8.8, ckan-2.9.5]
-        python-version: [py2, py3]
-        exclude:
-          - ckan-version: ckan-2.8.8
-            python-version: py3
+        ckan-version: [2.9, 2.9-py2, 2.8]
 
-    name: Continuous Integration build on CKAN ${{ matrix.ckan-version }} ${{ matrix.python-version }}
+    name: Continuous Integration build on CKAN ${{ matrix.ckan-version }}
     runs-on: ubuntu-latest
     container: integratedexperts/ci-builder
```

Comment:
You could investigate adding container caching to our github actions list to see if that makes it even faster (for base) https://github.com/marketplace/actions/docker-layer-caching